pkg/scaler/scaler.go

```diff
@@ -159,15 +161,14 @@ func (s *Scaler) Do(ctx context.Context) {
 
 	logger := s.logger
 
-	usage, err := s.client.GetCurrentUsage(ctx)
+	// get backend instances to calculate usage
+	ins, err := s.backend.Instances()
```

Comment:
should we store the instances in the `Scaler` struct? So they can be used later, in `sclaeDown` method for example or `removeNode`?

Reply:
instances is already passed to `removeNode` func no need to store it

Reply:
That what I mean, when you have instance method you can store the instances in the instance properties and avoid passing to the methoeds. This is just preferred (not enforced) in OOP.

Reply:
i"m calling in two places for `backend.Instances()` in `DO` and `GC` func they are in separate goroutines, so what i should do store instances obj only for `DO` and the `GC` will you use as now?

Reply:
Lets keep it as is, with go routins you will have to deal concurency.